src/autogluon/cloud/cluster/ray_aws_cluster_config_generator.py

```diff
@@ -101,21 +115,29 @@ def _update_config(
             The instance profile arn to be attched to the worker node. If not specified, a default one will be created with full EC2 and S3 permissions.
             To learn how to scope down the permission,
                 https://github.com/ray-project/ray/issues/9327
+        initialization_commands: Optional[List[str]], default = None
+            The initialization commands of the ray cluster.
+            If not specified, will contain a default ECR login command to be able to pull AG DLC image, i.e.
+                - aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin 763104351884.dkr.ecr.us-east-1.amazonaws.com
```

Comment:
Do we know which version of aws cli is installed. The command slightly differs between v1 and v2

Reply:
This is actually a good point. I think it is whatever being installed on ec2 by default. This command is being called before we enter the docker container.
When I look at the instruction in ECR repo, this is the only command that it shows for login. I assume it's ok for now.

Reply:
Please make sure the AMI used by the EC2 is security blessed

Reply:
Sounds good to me. Thanks for following up